metadata.rb

```diff
@@ -6,7 +6,7 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 source_url       'https://github.com/artem-sidorenko/chef-cups'
 issues_url       'https://github.com/artem-sidorenko/chef-cups/issues'
-version          '0.12.0'
+version          '0.13.0'
```

Comment:
```suggestion
version          '0.12.0'
```

@olqs may I ask you to remove the version bump? I'll do it by myself during the release procedure
```suggestion
version          '0.13.0'
```